package.json

```
@@ -56,12 +56,14 @@
     "undici-types": "6.x"
   },
   "scripts": {
-    "build": "rm -rf dist shims && run build:bundle && ts-node ./mkshims.ts",
+    "build": "run clean && run build:bundle && ts-node ./mkshims.ts",
     "build:bundle": "esbuild ./sources/_lib.ts --bundle --platform=node --target=node18.17.0 --external:corepack --outfile='./dist/lib/corepack.cjs' --resolve-extensions='.ts,.mjs,.js'",
+    "clean": "run rimraf dist shims",
     "corepack": "ts-node ./sources/_cli.ts",
     "lint": "eslint .",
     "prepack": "yarn build",
-    "postpack": "rm -rf dist shims",
+    "postpack": "run clean",
+    "rimraf": "node -e 'for(let i=2; i<process.argv.length;i++)fs.rmSync(process.argv[i],{recursive:true,force:true});'",
```

Small inconsistency in spacing here
```suggestion
    "rimraf": "node -e 'for(let i=2;i<process.argv.length;i++)fs.rmSync(process.argv[i],{recursive:true,force:true});'",
```
or
```suggestion
    "rimraf": "node -e 'for(let i=2; i<process.argv.length; i++) fs.rmSync(process.argv[i], {recursive:true, force:true});'",
```